ophyd/device.py

```diff
@@ -909,6 +909,10 @@ def walk_subdevice_classes(cls):
         '''
         for attr in cls._sub_devices:
             cpt = getattr(cls, attr)
+            if cpt is None:
+                # Subclasses can override this, making this None...
```

Comment:
This is a useful escape hatch. Good call.

Reply:
Do we still need this with https://github.com/NSLS-II/ophyd/pull/688 merged?

Reply:
It can be removed